test/unit/fixtures/form/claim.html

```diff
@@ -190,6 +190,8 @@
         <div class="text-wrapper field-insured-property-country1701932419758 field-wrapper" data-id="textinput-8b986fa9bb" data-required="false"><label
                 for="textinput-8b986fa9bb" class="field-label">Country (Optional)</label><input type="text"
                 id="textinput-8b986fa9bb" name="Insured_Property_Country1701932419758" autocomplete="off"></div>
+        <div class="button-wrapper field-reset1713344016774 field-wrapper reset-wrapper" data-id="reset-0d74964a9e">
```

Comment:
Can you help me understand why is this movement required in test, shouldn't it be a complete removal. We simply removed the reset and submit buttons from wizard button wrapper.

Reply:
Thanks @TalmizAhmed